website/Makefile

```diff
@@ -7,6 +7,7 @@ build:
 		--rm \
 		--tty \
 		--volume "$(shell pwd):/website" \
+		-e "ENV=production" \
```

Comment:
Hm we use `make website` for local testing. So this would use the production Segment ID?

Reply:
I don't think it would, but I could be mistaken. I made sure the `website` task below doesn't use this environment variable, only the "build" task 😁

Reply:
Totally thought it was under `website`. Ignore and LGTM